pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -131,6 +124,10 @@ def on_run_start(self, batch: Any, batch_idx: int, dataloader_idx: int):
         void(batch_idx, dataloader_idx)
         self._remaining_splits = list(enumerate(self._tbptt_split_batch(batch)))
 
+    def on_advance_start(self, *args: Any, **kwargs: Any) -> None:
+        super().on_advance_start(*args, **kwargs)
+        self.progress.increment_started()
```

Comment:
I think this is different here. 
`self.progress` is for the batch loop which iterates over batch splits. 

the one that you mean is in `training_epoch_loop` which iterates over batches in an epoch

Reply:
yep. 
the progress of a loop should be defined by how many advances we have made. otherwise what are we even tracking? how many calls to run() were made? that wouldn't be useful at all, and counter intuitive.